readthedocs/projects/admin.py

```diff
@@ -325,6 +327,42 @@ class FeatureAdmin(admin.ModelAdmin):
     filter_horizontal = ('projects',)
     readonly_fields = ('add_date',)
     raw_id_fields = ('projects',)
+    actions = ['assign_to_sampled_random_projects']
+
+    def assign_to_sampled_random_projects(self, request, queryset):
+        """Assign features to 'N' sampled random projects."""
+        random_projects = None
+        if 'apply' in request.POST:
+            form = RandomProjectsInputForm(request.POST)
+            if form.is_valid():
+                n = form.cleaned_data.get('number')
+
+                # Taking first 'n' random projects.
+                random_projects = Project.objects.order_by('?')[:n]
```

Comment:
I think we need to avoid spam projects here with a query like:

    Project.objects.filter(users__profile__banned=False)

Otherwise, it's probable that a few real projects get assigned the feature flag we want.

Reply:
Thank you.
I have pushed the changes.